### ISSUE TITLE
New placeholder cache type for own stuff in copy to clipboard menu in cache listing #GCType#.

### ISSUE THREAD

**Issue by frenkovo** [2Abendsegler/GClh#1327]
Hi, will the cache type also be implemented? 

#GCType#

Thank you very much, great work ;)

**Comment by 2Abendsegler** [comment]
@frenkovo  
#GCType# is implemented and comes with version 0.10.6.
You can implement it in advance [here](https://github.com/2Abendsegler/GClh/raw/collector/gc_little_helper_II.user.js).
You can back [here](https://github.com/2Abendsegler/GClh/raw/master/gc_little_helper_II.user.js), if you want.

**Comment by 2Abendsegler** [comment]
Close

---
```
	<li>
		<strong>New:</strong> New placeholder cache type for own stuff in copy to clipboard menu in cache listing #GCType#. [<a href="https://github.com/2Abendsegler/GClh/issues/1327" title="Issue 1327">1327</a> / <a href="https://www.geocaching.com/profile/?u=2Abendsegler" title="Thanks to 2Abendsegler">2Abendsegler</a>]<br>
		<a href="https://www.geocaching.com/my/#GClhShowConfig#a#settings_show_copydata_menu" title="Link to your GClh Config">Settings -> Listing<br><br>
	</li>
```
